### ISSUE TITLE
Is this work for Arduino?

### ISSUE THREAD

**Issue by AllwinAristo** [GadgetReboot/Arduino#2]

**Comment by Lauwy222** [comment]
Yes, this is the [video](https://www.youtube.com/watch?v=KqBruMXQjmA) he made using the [Arduino UNO](https://store.arduino.cc/products/arduino-uno-rev3/).
